depgenerators/maven.req

```diff
@@ -299,7 +299,11 @@ class TagBuilder(object):
     def _normalize_java_requires(self, req):
```

Comment:
In the `_get_java_requires` function above, the versions are compared as strings, which can produce incorrect results. (`"9" > "10"`)

Reply:
Good catch, I'll add test for it and fix it.